benchmarks/src/main/java/io/druid/benchmark/query/GroupByBenchmark.java

```diff
@@ -116,7 +116,7 @@
   @Param({"100000"})
   private int rowsPerSegment;
 
-  @Param({"basic.A"})
+  @Param({"nested.A"})
```

Comment:
This will disable the "basic.A" query and schema from running, you can enable them both:

@Param({"basic.A", "nested.A"})